packages/demo-react/src/index.jsx

```diff
@@ -1,5 +1,5 @@
 import ReactDOM from 'react-dom';
 import React from 'react';
-import App from './App';
+import App from './App.jsx';
```

Comment:
jsx extensions are still automatically resolved. In this case there is also an App.scss, so the extension is required.